jaxley/modules/base.py

```diff
@@ -876,7 +876,7 @@ def data_set(
         self,
         key: str,
         val: Union[float, jnp.ndarray],
-        param_state: Optional[List[Dict]],
+        param_state: Optional[List[Dict]] = None,
```

Comment:
I would prefer to not have `None` default here, as it might make the following bug more frequent:
```python
def sim(params):
    pstate = cell.data_set("radius", 2.0)
    pstate = cell.data_set("length", 20.0)
    # ....
```